src/history/HistoryArchiveReportWork.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+// Copyright 2020 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "history/HistoryArchive.h"
```

Comment:
include this in HistoryArchiveReportWork.cpp, and forward declare in this file.

Reply:
Fixed!